### ISSUE TITLE
how to play ai?

### ISSUE THREAD

**Issue by cloudjanak** [jreinstra/Scrabble.py#1]
is it possiable like below in web-site
pass the current board and rack tiles and it will return all playable words with starting position and direction.

http://www.lexicalwordfinder.com/?game=Words_With_Friends&layout=1&tiles=gtsfsdf&board=----------------------------------------------------------------------------------------------------------------hello------------------------------------------------------------------------------------------------------------&lexicon=wwf

**Comment by jreinstra** [comment]
Yes, that is possible.  In fact, this program does something very similar, except is designed more like a game.

In order to play the AI, you'll need to run the python program.

**Comment by cloudjanak** [comment]
can you add one example how to run the program 
like what we have to pass in parameter?

**Comment by jreinstra** [comment]
The program has a graphics interface and does not accept command-line parameters.  Just use the command "python /file/path/scrabble.py" and then use the GUI to add some tiles.  Click the "cheat" button and the program will find the best possible word and position using the given tiles and place it.

**Comment by cloudjanak** [comment]
so its not possible to pass direct board and tile ??

**Comment by jreinstra** [comment]
That's correct.  I might add that capability at some point, however.

**Comment by cloudjanak** [comment]
so can you please help me by adding that capability??

**Comment by cloudjanak** [comment]
one simple method where we can pass two parameter tiles on hand and board thats it .

**Comment by jreinstra** [comment]
Sure, I'll work on it.

**Comment by cloudjanak** [comment]
i am eagerly waiting for that..!!
Thank you in advance.
